### PR TITLE
readme file is change

### PR DESCRIPTION
**Pull request by RaghavaIT**
it is not related to standard code instruction

# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by RaghavaIT** [comment]
changes done

**Comment by RaghavaIT** [comment]
change done

File: README.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ See the [**Contribution guide**](/1-CONTRIBUTION-GUIDE/README.md#contribution-gu
 
 ## NOTE
 
+
 We have finished migration of the samples to subfolders, see [azure.com](https://azure.microsoft.com/en-us/documentation/templates) if you need help finding a sample.  A few obsolete samples were removed in the migration.
 
 ### Final Note
-
+## this is module for test purpose
```
Comment (bmoore-msft) [inline README.md:13]: This is a header - what is the text needed below the header?

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the feedback.

**Pull request closed without merging**